### PR TITLE
Promote publication

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
Updates the content on distributing/publishing collections to make distribution options clearer. Per conversation with @thaumos. 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

##### ADDITIONAL INFORMATION
You can read the proposed new content on our [testing site](http://docs.testing.ansible.com/ansible/2.11/dev_guide/developing_collections.html#distributing-collections) temporarily - the testing site changes constantly and content may change or disappear without notice!.

### CODE REVIEW

**Review by bmillemathias — APPROVED** [review]

**Comment by acozine** [comment]
rebuild

**Review by thaumos — APPROVED** [review]

File: docs/docsite/rst/dev_guide/developing_collections.rst [hunk 1]
```diff
@@ -417,47 +418,59 @@ You can also test a version of your collection in development by installing it f
...
-----------------------
+Distributing collections
+========================
+
+You can distribute your collections by publishing them on a distribution server. Distribution servers include Ansible Galaxy, Red Hat Automation Hub, and privately hosted Automation Hub instances. You can publish any collection to Ansible Galaxy and/or to a privately hosted Automation Hub instance. If your collection is certified by Red Hat, you can publish it to the Red Hat Automation Hub.
+
+Prerequisites
+-------------
+
+1. Get a namespace on each distribution server you want to use (Galaxy, private Automation Hub, Red Hat Automation Hub)
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/developing_collections.rst]:
```suggestion
1. Get a namespace on each distribution server you want to use (Galaxy, private Automation Hub, Red Hat Automation Hub).
```

File: docs/docsite/rst/dev_guide/developing_collections.rst [hunk 2]
```diff
@@ -417,47 +418,59 @@ You can also test a version of your collection in development by installing it f
...
+2. Get an API token for each distribution server you want to use
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/developing_collections.rst]:
```suggestion
2. Get an API token for each distribution server you want to use.
```

File: docs/docsite/rst/dev_guide/developing_collections.rst [hunk 3]
```diff
@@ -417,47 +418,59 @@ You can also test a version of your collection in development by installing it f
...
+3. Specify your API token(s)
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/developing_collections.rst]:
```suggestion
3. Specify your API token(s).
```

File: docs/docsite/rst/dev_guide/developing_collections.rst [hunk 4]
```diff
@@ -417,47 +418,59 @@ You can also test a version of your collection in development by installing it f
...
 ^^^^^^^^^^^^^^^^^^^^^^
 
-To upload your collection to Galaxy, you must first obtain an API token (``--token`` in the ``ansible-galaxy`` CLI command or ``token`` in the :file:`ansible.cfg` file under the ``galaxy_server`` section). The API token is a secret token used to protect your content.
+You need an API token for Galaxy and/or Automation Hub to upload your collection. Use the API token(s) to authenticate your connection to the distribution server(s) and protect your content.
 
 To get your API token:
 
 * For Galaxy, go to the `Galaxy profile preferences <https://galaxy.ansible.com/me/preferences>`_ page and click :guilabel:`API Key`.
-* For Automation Hub, go to https://cloud.redhat.com/ansible/automation-hub/token/ and click :guilabel:`Load token` from the version dropdown.
+* For Automation Hub, go to `the token page <https://cloud.redhat.com/ansible/automation-hub/token/>`_ and click :guilabel:`Load token` from the version dropdown.
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/developing_collections.rst]:
```suggestion
* For Automation Hub, go to `the token page <https://cloud.redhat.com/ansible/automation-hub/token/>`_ and click :guilabel:`Load token`.
```

Comment (samccann) [reply docs/docsite/rst/dev_guide/developing_collections.rst]: Just looked and there's no version dropdown, just a button labeled load token.

**Review by samccann — COMMENTED** [review]
some nits

File: docs/docsite/rst/dev_guide/developing_collections.rst (hunk 4 above)
Comment (acozine) [reply docs/docsite/rst/dev_guide/developing_collections.rst]: Better and better!

**Review by samccann — APPROVED** [review]

**Pull request merged**